### PR TITLE
feat: migration page

### PR DESCRIPTION
**Pull request by Tulun**
Story details: https://app.shortcut.com/uma-project/story/5498/across-v2-migration-page

### CODE REVIEW

File: src/components/PoolSelection/PoolSelection.tsx [hunk 1]
```diff
@@ -56,6 +63,22 @@ const PoolSelection: FC<Props> = ({ token, setToken, tokenList, chainId }) => {
   return (
     <AnimatePresence>
       <Wrapper>
+        {showMigrationPage ? (
```
Comment (daywiss) [inline src/components/PoolSelection/PoolSelection.tsx:66]: this already exists, there is an env which enables this, we should keep this so we can enable/disable with the env.

image

Imo we should use this copy in the existing migration warning

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -13,12 +13,15 @@ import {
...
-const LINKS = [
-  { href: "/", name: "Bridge" },
-  { href: "/pool", name: "Pool" },
-  { href: "/transactions", name: "Transactions" },
-];
+const LINKS = !showMigrationPage
+  ? [
+      { href: "/", name: "Bridge" },
+      { href: "/pool", name: "Pool" },
+      { href: "/about", name: "About" },
```
Comment (daywiss) [inline src/components/Header/Header.tsx:22]: this should be transactions, not about

Comment (Tulun) [reply src/components/Header/Header.tsx:22]: fixed

**Comment by mrice32** [comment]
Screen Shot 2022-05-16 at 5 30 59 PM
@Tulun something odd I noticed in the v2 UI (maybe unrelated to this PR): when I scrolled down, I was able to scroll past the bottom of the box. This doesn't happen in the v1 UI.

**Review by mrice32 — APPROVED** [review]
Looks great! I ran this locally and noticed a minor difference between this and the v1 UI, but not sure if it's related to these changes: https://github.com/across-protocol/frontend-v2/pull/88#issuecomment-1128158741.

**Comment by Tulun** [comment]
> Screen Shot 2022-05-16 at 5 30 59 PM
> 
> @Tulun something odd I noticed in the v2 UI (maybe unrelated to this PR): when I scrolled down, I was able to scroll past the bottom of the box. This doesn't happen in the v1 UI.

Should be unrelated, but we can make a bug ticket for it.

**Pull request merged**